src/functions-and-arrays.js

```diff
@@ -26,13 +68,37 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersAvg) {
```

Comment:
Here you don't need the variable `average` you can return the `sum` divided by the `length` of the array:

```
function averageNumbers(numbersAvg) {
  let sum = 0;

  if (!numbersAvg.length) {
    return null;
  }

  for (let i = 0; i < numbersAvg.length; i++) {
    sum += numbersAvg[i];
  }

  return sum / numbersAvg.length;
}
```